HEC.FDA.Model/compute/ImpactAreaScenarioSimulation.cs

```diff
@@ -1121,6 +1126,7 @@ public SimulationBuilder withFlowFrequency(ContinuousDistribution continuousDist
             public SimulationBuilder withFlowFrequency(GraphicalUncertainPairedData graphicalUncertainPairedData)
             {
                 _simulation._frequency_discharge_graphical = graphicalUncertainPairedData;
+                _simulation.AddSinglePropertyRule("flow frequency", new Rule(() => { _simulation._frequency_discharge_graphical.Validate(); return !_simulation._frequency_discharge_graphical.HasErrors; }, string.Join(Environment.NewLine, _simulation._frequency_discharge_graphical.GetErrors())));
```

Comment:
Understood, thank you. 

Ultimately, I removed this property rule because we don't have any validation identified on graphical frequency functions.